aggregating-profiler/src/main/java/ru/fix/aggregating/profiler/RegexpTagger.java

```diff
@@ -0,0 +1,39 @@
+package ru.fix.aggregating.profiler;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.HashMap;
+import java.util.regex.Pattern;
+
+/**
+ *
+ * @author Andrey Kiselev
+ */
+
+public class RegexpTagger implements Tagger {
+    private final String defaultTagName;
```

Comment:
Maybe rename defaultTagName to simply tagName ?

Reply:
ok